chat_server/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "portail-chat",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "start": "tsc && node dist/index.js",
+    "test": "tsc && mocha ./dist/test.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/chai": "^4.2.7",
+    "@types/express": "^4.17.1",
+    "@types/jsonwebtoken": "^8.3.5",
+    "@types/mocha": "^5.2.7",
+    "@types/pg": "^7.11.2",
+    "@types/socket.io": "^2.1.4",
+    "@types/socketio-jwt": "0.0.0",
+    "chai": "^4.2.0",
+    "jsonwebtoken": "^8.5.1",
+    "mocha": "^6.2.2",
+    "mongodb": "^3.4.1",
+    "mongodb-memory-server": "^6.1.1",
```

Comment:
Why do we need mongodb ?

Reply:
Deprecated import, fixing it and merging master